### PR TITLE
Added chart to run the lyra runs cleanup task

### PR DESCRIPTION
**Pull request by ArtieReus**
We need a sort of cron job to run a cleanup task and remove old automation runs.

### CODE REVIEW

**Comment by ArtieReus** [comment]
@databus23 lgty?

File: openstack/lyra/templates/runs-pruning.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+kind: Deployment
+apiVersion: extensions/v1beta1
+
+metadata:
+  name: runs-pruning
```
Comment (databus23) [inline openstack/lyra/templates/runs-pruning.yaml]: lets call this something more generic: db-pruner (also rename the file)

File: openstack/lyra/templates/runs-pruning.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+spec:
+  revisionHistoryLimit: 5
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: runs-pruning
```
Comment (databus23) [inline openstack/lyra/templates/runs-pruning.yaml]: db-pruner

File: openstack/lyra/templates/runs-pruning.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    spec:
+      containers:
+        - name: pruning
```
Comment (databus23) [inline openstack/lyra/templates/runs-pruning.yaml]: pruner

File: openstack/lyra/templates/runs-pruning.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            - |
+              while true; do
+                if [ $(date '+%H%M') = '{{ .Values.session_cleanup_time }}' ]; then
+                  bin/rake run:cleanup
+                  echo "bin/rake run:cleanup exited"
+                fi
+                sleep 55
+              done
+          env:
+{{ include (print .Template.BasePath "/_env.tpl") . | indent 12 }}
```
Comment (databus23) [inline openstack/lyra/templates/runs-pruning.yaml]: Has to be `{{ include "env.vars" . | indent 12 }}`. See api-deployment.yaml

File: openstack/lyra/templates/runs-pruning.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
+kind: Deployment
+apiVersion: extensions/v1beta1
+
+metadata:
+  name: runs-pruning
+
+spec:
+  revisionHistoryLimit: 5
```
Comment (databus23) [inline openstack/lyra/templates/runs-pruning.yaml]: Make it `3`

File: openstack/lyra/templates/runs-pruning.yaml (hunk 1 above)
Comment (ArtieReus) [reply openstack/lyra/templates/runs-pruning.yaml]: ok

File: openstack/lyra/templates/runs-pruning.yaml (hunk 4 above)
Comment (ArtieReus) [reply openstack/lyra/templates/runs-pruning.yaml]: right! i was missing something I couldn't remember...

**Pull request merged**